package.json

```diff
@@ -82,7 +82,7 @@
     "jest-enzyme": "^7.0.0",
     "jest-styled-components": "^6.2.2",
     "lerna": "^3.10.5",
-    "prettier": "^1.13.4",
+    "prettier": "^1.16.4",
```

Comment:
please update it on `pre-commit-config.yaml` as well

Reply:
on `pre-commit-config.yaml` we already have `rev: "1.16.4"` for prettier 🤔

Reply:
Oh, sorry! I must have opened the webapp file which is 1.13.6